website/docs/using-unleash/deploy/license-keys.mdx

```diff
@@ -1,39 +1,41 @@
 ---
-title: License Keys for Unleash Enterprise (BETA)
+title: License Keys for Unleash Enterprise
 ---
 
-import VideoContent from '@site/src/components/VideoContent.jsx'
+import VideoContent from "@site/src/components/VideoContent.jsx";
 
 This document describes all aspects and workflows of license key management in Unleash 5.8 and above.  
```

Comment:
I think we should mention that this is available from 5.8 and required for self hosted enterprise after v6

Reply:
Good call!